### PR TITLE
Reporting folder start

### PR DESCRIPTION
**Pull request by pld**
* creates a `reporting folder for the reporting docker-compose file and all reporting templates
* moves the existing `nifi` NiFi templates folder to the `reporting` folder
* creates a `reporting/docker-compose.yml` to start adding containers for reporting
* adds the a `nifi` entry to the `docker-compose.yml` file
* adds an environment variable to `.env` used by the `nifi` entry of the `docker-compose.yml` file

### CODE REVIEW

**Comment by craigappl** [comment]
@chunky56 requesting your review because you originally setup Nifi in this repo.

File: reporting/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+version: "0.1"
```
Comment (chunky56) [inline reporting/docker-compose.yml]: I'm curious; is there a reason to use version 0.1 here?

Comment (pld) [reply reporting/docker-compose.yml]: not really, seemed like a reasonable number to start w/, but I'm not familiar w/what the best practice is and happy w/whatever that is

Comment (chunky56) [reply reporting/docker-compose.yml]: Ah okay, docker-compose files start at version 1, and all our compose files are at least version 2. Probably best to match the main file in the Ref Distro, 3.3.

https://docs.docker.com/compose/compose-file/compose-versioning/

**Pull request merged**